src/averageAges.js

```diff
@@ -55,7 +64,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = people.filter(person => person.mother !== undefined);
```

Comment:
If you have a case that is a person.mother: null?)
```suggestion
  const children = people.filter(person => !!person.mother);
```